README.md

```diff
@@ -13,3 +13,28 @@ This is a simple blogging engine with a few basic features:
 This means if you copy-paste code from this sample you are likely opening yourself up to various security holes, bad practice,
 and bad design. It's a demonstration of the surface API of Authboss and how the library can be used to make a functioning web
 project, to use this sample as anything else is malpractice.
+
+# Getting Started
+
+To get started, clone the repo and execute the following commands
+
+```
+cd authboss-sample
+export GOPATH=`pwd`

```

Comment:
This isn't typical Go workflow. We don't use Godeps or Glide which would require this. As such I think the instructions should be modified to reflect that.